### PR TITLE
Add runtime info to Bowtie report

### PR DESCRIPTION
**Pull request by AgniveshChaubey**
### What is this PR for?
This PR displays runtime information (including `os`, `os_version`, `language` and `language_version`) to Bowtie Report!
Here bootstrap **_modal_** is used to to display the above additional information.

### CODE REVIEW

**Comment by AgniveshChaubey** [comment]
Hi @Julian, Please have a look at this PR!

**Comment by Julian** [comment]
I've seen it and will have a look in a bit, no need to re-ping until at least a day or two has gone by, though I certainly hope to have feedback before then!

**Comment by AgniveshChaubey** [comment]
I'm sorry for this! I thought you missed the notification, that's why... Will take care from now onwards :)

File: bowtie/templates/report.html.j2 [hunk 1]
```diff
@@ -190,6 +190,10 @@
                   </th>
                   <td>
                     <small class="font-monospace text-muted">{{ implementation.get("version", "") }}</small>
+
+                    <button type="button" class="btn btn-sm btn-secondary" data-bs-toggle="modal" data-bs-target="#implementation-{{ loop.index }}-runtime-info">
+                      more info
```
Comment (Julian) [inline bowtie/templates/report.html.j2:195]: Let's maybe make this even more conspicuous with one of those [i icons](https://icons.getbootstrap.com/icons/info-circle-fill/). Make sense?

File: bowtie/templates/report.html.j2 [hunk 2]
```diff
@@ -363,6 +367,29 @@
...
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="implementation-{{ loop.index }}-runtime-info-label">
+                <b>{{ implementation.name }} </b> <small class="text-muted"> {{ implementation.language }}</small>
+              </h5>
+              <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <p><strong>OS Version : </strong>{{ implementation.get("os_version", "") }}</p>
```
Comment (Julian) [inline bowtie/templates/report.html.j2:382]: There are extra spaces in front of the `:`'s here, can you trim those.

Also maybe let's only show values where they're non-empty (so hide any row where we don't have a value for that key).

**Review by Julian — CHANGES_REQUESTED** [review]
No worries at all! Left you two minor comments which maybe you can address, but think this seems pretty good to me! Let me know what you think.

File: bowtie/templates/report.html.j2 (hunk 1 above)
Comment (AgniveshChaubey) [reply bowtie/templates/report.html.j2]: Sure!

File: bowtie/templates/report.html.j2 (hunk 2 above)
Comment (AgniveshChaubey) [reply bowtie/templates/report.html.j2]: Extra white space is removed!
Conditionally rendering the row if its value is non-empty

**Comment by Julian** [comment]
Nice! Looks great, thanks again @AgniveshChaubey!

**Pull request merged**